sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/DataLoader.java

```diff
@@ -71,11 +73,13 @@ private void bulkCreateItems(final Map<Key, ObjectNode> records) {
             database.getId(),
             containerName);
 
+        // We want to wait longer depending on the number of documents in each iteration
+        final Duration blockingWaitTime = Duration.ofSeconds(120 * (_configuration.getBulkloadBatchSize() / 200000));
```

Comment:
This is int value , it will round to lower value ,how about rounding to upper value . For e.g. 500000/200000 is 2, but it will be better if we do 3. Just to be extra safe Thoughts?

Reply:
Going to go with the -1 and + 1 change.